src/Factories/FunFunFactory.php

```diff
@@ -354,7 +355,10 @@ public function getEntityManager(): EntityManager {
 	}
 
 	private function newDonationEventLogger(): DonationEventLogger {
-		return new DoctrineDonationEventLogger( $this->getEntityManager() );
+		return new BestEffortDonationEventLogger(
```

Comment:
Think this name is fine. I probably would have gone with FallbackDonationEventLogger